### ISSUE TITLE
F*ck The BMW

### ISSUE THREAD

**Issue by VyseM4H** [Hurstzy/ShorelineRoleplay#222]
**Why do you purpose this balance?**
[a thorough description for your balance request *remove line to answer*]
The BMW has become a cornerstone of trash rp and terrible driving. I'm not even an officer anymore but cant describe how I feel when I consistantly see BMWs burning out around corners and going Mach 2 to flee from LEOs. Have even been in a car with someone who wanted to run "Because I know I can get away". Which is yikes.

**What is your solution?**
[a thorough description for your balance solution *remove line to answer*] Stop putting it on sale at PDM let the remaining ones be whats left in the city.

**Other info**
[any other info? *remove line to answer*]
Nah... Just f*ckem in general ya know?

**Comment by Jamesmayl** [comment]
As a cop this is 100% true

**Comment by DerekStealth** [comment]
**ON GOD** , bmw is soo annoying

**Comment by factorAD** [comment]
big facts.
It was out of stock at the PDM last time I checked though

**Comment by VyseM4H** [comment]
@factorAD  im saying remove it from own entirely those that have them keep them or sell an to another person

**Comment by factorAD** [comment]
> @factorAD im saying remove it from PDM entirely those that have them keep them or sell em to another person

oh I was under the impression that "out of stock" was psuedo-removal . I agree, remove it

**Comment by Hurstzy** [comment]
Would a balance be more appropriate, we can nerf/buff any car.

**Comment by factorAD** [comment]
> Would a balance be more appropriate? We can nerf/buff any car.

yes, they are just unrealistic at the moment, and they are overly common to just discontinue without also nerfing imo

**Comment by Jamesmayl** [comment]
Yes
